packages/expo-ui/src/swift-ui/Layout/index.tsx

```diff
@@ -9,7 +9,7 @@ type TapEvent = ViewEvent<'onTap', object> & {
 };
 
 interface StackBaseProps extends CommonViewModifierProps {
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
this was done as i needed a simple Box for animation examples 😅. Maybe we add support for [Rectangle](https://developer.apple.com/documentation/swiftui/rectangle)